tests/test_northup.py

```diff
@@ -6,7 +6,7 @@
 from matplotlib import pyplot as plt
 import numpy as np
 
-def test_northup(save_mock_dataset=False,save_derot_dataset=False,save_comp_figure=False):
+def test_northup(save_mock_dataset=False,save_derot_dataset=False,save_comp_figure=False,test_offset=False):
```

Comment:
keywords actually don't work in pytest automated testing (at least in how we set it up). You should just make a new test to test your offset.

Reply:
it'll be clearing then when the assert fails, why it failed. Right now, it's unclear if the assert fails, whether that's due to your changes or due to issues in the northup step function.